pkg/upapi/ep_outages.go

```diff
@@ -1,19 +1,11 @@
-package uptime
+package upapi
```

Comment:
api?

Reply:
Calling exported package `api` seems way too generic. I often find myself aliasing such packages like

```
import uptimeapi github.com/uptime-client-go/pkg/api
```

I mean, in imports package name are properly namespaced. In later code they are not and `api` might collide with other imports. Aliasing addresses that issue to some degree. But prefixing package name at lower level addresses it even better because it save braicycles. And again, at uptime.come we are lucky having such a nice convenient 2-letter prefix we use everywhere. Why not use it here?